CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+### Breaking changes
+
+#### Check your favicons, app icons and OpenGraph image still work
+
+We've changed the names, formats and sizes of icon assets that we distribute in Frontend. You will want to check that the correct files are in place.
+
+The following files have been added to the assets folder:
```

Comment:
I think we need to call out `assetPath` (which defaults to `/assets`), `assetUrl` and `opengraphImageUrl`

Could be helpful to refer to the link in https://github.com/alphagov/govuk-frontend/pull/4445/files#r1389317268 earlier but see what @claireashworth recommends

Reply:
Can do this. Another option is to move the file path line up between current lines 11 and 13, as it's an action users need to do

Reply:
Was thinking more for context as lots of services use project-specific paths. When you read "assets folder" it doesn't necessarily help identify where the images are copied or deployed

>If a user is already using the Nunjucks template (and thus `assetPath`, et al.), then they are will already have these variables configured. They don't need to be changed if the images are being replaced in-place.

Even the Nunjucks templates can override them elsewhere:

```mjs
{
  assetPath: "/specific/directory/that/other/team/manages",
  assetUrl: "http://cdn-assets-here.domain.com/assets",
  opengraphImageUrl: "http://social-media-contract-assets-here.domain.com/assets/specific.png"
}
```

Reply:
Do we still think specifically calling it out is needed now that documentation relating to it is linked from the changelog entry?

Reply:
I think it should be ok as is. We can always amend it if we get feedback to the contrary.

Reply:
Nice, thanks for sticking the link in. Yeah we're all set now 🙌